scripts/exgdas_enkf_update.sh

```diff
@@ -213,7 +213,7 @@ for imem in $(seq 1 $NMEM_ENS); do
    for FHR in $nfhrs; do
       ${NLN} "${COMIN_ATMOS_HISTORY_MEM_PREV}/${GPREFIX}atmf00${FHR}${ENKF_SUFFIX}.nc" \
          "sfg_${PDY}${cyc}_fhr0${FHR}_${memchar}"
-      if [ $GSI_SOILANAL = "YES" ]; then
+      if [[ $DO_GSISOILDA = "YES" ]]; then
```

Comment:
```suggestion
      if [[ "${DO_GSISOILDA}" = "YES" ]]; then
```

Reply:
The shellcheck linter check does not capture this because we are running a differential check; meaning if the error existed before and after the modification, it gets cancelled out and does not get reported here. Until we remove all instances of these reports, we will have to continue running a differential checker.  This is just a small step towards getting there.  Hope this explains the pesky nature of these suggested changes.